hircluster.c

```diff
@@ -3708,13 +3708,20 @@ redisClusterAsyncContext *redisClusterAsyncContextInit(void) {
     return acc;
 }
 
+/* Creates an asynchronous context with a prepared slot map containing
+ * configured nodes. Since this API is called before any event system can be
+ * attached there is no immediate connection establishment performed.
+ * Users of this API is required to attach an event engine adapter to the
```

Comment:
is → are

Btw, do we put the documentation in the .h file in this project? It seems a bit random. Some are documented in the .h file and some in the .c file already... And some in README.